client/state/data-layer/wpcom/sites/users/index.js

```diff
@@ -38,60 +38,51 @@ export const normalizeUser = user =>
 /**
  * Dispatches a request to fetch post revisions users
  *
- * @param {Function} dispatch Redux dispatcher
  * @param {Object} action Redux action
  */
-export const fetchUsers = ( { dispatch }, action ) => {
+export const fetchUsers = action => {
 	const { siteId, ids, page = 1, perPage = DEFAULT_PER_PAGE } = action;
-	dispatch(
-		http(
-			{
-				path: `/sites/${ siteId }/users`,
-				method: 'GET',
-				apiNamespace: 'wp/v2',
-				query: {
-					include: ids,
-					page,
-					per_page: perPage,
-				},
+	return http(
+		{
+			path: `/sites/${ siteId }/users`,
+			method: 'GET',
+			apiNamespace: 'wp/v2',
+			query: {
+				include: ids,
+				page,
+				per_page: perPage,
 			},
-			action
-		)
+		},
+		action
 	);
 };
 
 /**
  * Dispatches returned users
  *
- * @param {Function} dispatch Redux dispatcher
  * @param {Object} action Redux action
  * @param {Array} users raw data from post revisions API
  */
-export const receiveSuccess = ( { dispatch }, action, users ) => {
+export const receiveSuccess = ( action, users ) => {
 	const { page = 1, perPage = DEFAULT_PER_PAGE } = action;
 	const normalizedUsers = map( users, normalizeUser );
 
 	if ( get( getHeaders( action ), 'X-WP-TotalPages', 0 ) > page ) {
-		fetchUsers(
-			{ dispatch },
-			{
-				...omit( action, 'meta' ),
-				page: page + 1,
-				perPage,
-			}
-		);
+		fetchUsers( {
+			...omit( action, 'meta' ),
+			page: page + 1,
+			perPage,
+		} );
 	}
 
-	map(
-		normalizedUsers,
-		flow(
-			receiveUser,
-			dispatch
-		)
-	);
+	return map( normalizedUsers, flow( receiveUser ) );
```

Comment:
I think we can remove the `flow()` call here because we're not composing functions anymore.